pkgs/development/python-modules/mathutils/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "mathutils";
+  version = "3.3.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-uTca+BSfPz8F7hZctYOy33s/vJk1QG0cUFpSdrjlY68=";
+  };
+
+  pythonImportsCheck = [ "mathutils" ];
+
+  meta = with lib; {
+    description = "Utilities for working with datetime objects.";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

must not end in period

Reply:
This was completely wrong :-O I've put the correct description.